tools/SetupFlow/DevHome.SetupFlow/Models/WingetConfigure/SDKOpenConfigurationSetResult.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DevHome.SetupFlow.Common.Helpers;
+using DevHome.SetupFlow.Exceptions;
+using DevHome.SetupFlow.Services;
+using SDK = Microsoft.Windows.DevHome.SDK;
+
+namespace DevHome.SetupFlow.Models.WingetConfigure;
+
+public class SDKOpenConfigurationSetResult
+{
+    private readonly ISetupFlowStringResource _setupFlowStringResource;
+
+    public SDKOpenConfigurationSetResult(SDK.OpenConfigurationSetResult result, ISetupFlowStringResource setupFlowStringResource)
+    {
+        Result = result;
+        _setupFlowStringResource = setupFlowStringResource;
+
+        if (Result != null)
+        {
+            Field = result.Field.Clone() as string;
+            Line = result.Line;
+            Column = result.Column;
+            Value = result.Value.Clone() as string;
+            ResultCode = Result.ResultCode;
+            Succeeded = ResultCode != null && ResultCode.HResult == 0;
```

Comment:
I think you had it right in SDKApplyConfigurationSetResult, if ResultCode is null it's succeeded:
(ResultCode == null || ResultCode?.HResult == 0)

Reply:
updated but will probably need to update it again once we have sdk changes into main